### DISCUSSION TITLE
Unable to connect to Obsidian on Edge

### DISCUSSION THREAD

**Discussion by githubiwicyy** [coddingtonbear/obsidian-web#156] [Exotic Browser Support] [answered]
![1](https://github.com/coddingtonbear/obsidian-web/assets/48347628/87eda0da-16a7-4d0c-a44b-82337b5555cb)
help!!!!

**Comment by coddingtonbear** [comment]
You probably have another red exclamation point on the configuration page somewhere; I'm curious what it says when you hover over it or click on it.

Other than that, folks can't answer a whole lot without more information about your environment -- at the very, very least the browser and operating system you're running on.

**Reply by githubiwicyy** [reply]
Does the red exclamation point refer to this? 
 My operating system is windows, the browser I use is edge, and the plug-in I use is downloaded from the following webpage: https://chromewebstore.google.com/?hl=zh$$ 
I checked my API key and hostname
 Anyway, thanks for your reply
![image](https://github.com/coddingtonbear/obsidian-web/assets/48347628/a729b3ba-e3f1-4618-9fc0-88ae9accc7b4)

**Reply by coddingtonbear** [reply]
Yes-- that's one of them!  That particular exclamation point isn't all that helpful, though, unfortuantely.  There can sometimes be a slightly more helpful one next to the input field immediately above that, but at least you're not seeing one there.

That said, I'm afraid I can only help troubleshooting on Chrome and have no idea how to help on other browsers.  I'm honestly surprised at how many folks are using Chrome extensions on somewhat-chrome-compatible browsers like Edge.  Maybe if you search around in the discussions to see if you can find some tips, though?

**Reply by githubiwicyy** [reply]
Ok ,thank you very much~ although it is not solved.

**Reply by githubiwicyy** [reply, marked as answer]
I think I have found a solution. I need to download the certificate in the screenshot and open the installation directly, and then refresh the settings options page of Obsidian Web.
![1705418874901](https://github.com/coddingtonbear/obsidian-web/assets/48347628/48479dd7-f621-44a7-8298-ee34cb0c5c3a)
